examples/discovery-search-app/package.json

```diff
@@ -9,9 +9,10 @@
   "scripts": {
     "build": "cross-env SKIP_PREFLIGHT_CHECK=true react-scripts build",
     "eject": "cross-env SKIP_PREFLIGHT_CHECK=true react-scripts eject",
-    "start": "cross-env SKIP_PREFLIGHT_CHECK=true react-scripts start",
+    "start:client": "cross-env SKIP_PREFLIGHT_CHECK=true react-scripts start",
+    "start": "concurrently -n 'SERVER,CLIENT' -c 'blue,green' \"yarn run server:run\" \"yarn run start:client\" -k",
     "lint": "eslint --quiet '{src,cypress}/**/*.{js,jsx,ts,tsx}' './*.{js,jsx,ts,tsx}'",
-    "test:e2e": "cross-env BROWSER=none CYPRESS_baseUrl=http://localhost:4000 start-server-and-test server:run http://localhost:4000 'cypress run'",
+    "test:e2e": "cross-env BROWSER=none CYPRESS_baseUrl=http://localhost:3000 start-server-and-test start:client http://localhost:3000 'cypress run'",
```

Comment:
this is good, but if you take the suggestion above, it would just be `start` instead of `start:client`